src/components/ImportAccountsDrawer/ImportAccountsDrawer.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  Button,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  forwardRef,
+  Skeleton,
+  Switch,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Tooltip,
+  Tr,
+} from '@chakra-ui/react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { useIsFetching, useQueryClient } from '@tanstack/react-query'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { reactQueries } from 'react-queries'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { RawText, Text } from 'components/Text'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { accountIdToLabel } from 'state/slices/portfolioSlice/utils'
+import {
+  selectFeeAssetByChainId,
+  selectPortfolioCryptoPrecisionBalanceByFilter,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export type ImportAccountsDrawerProps = {
+  chainId: ChainId
+  isOpen: boolean
+  onClose: () => void
+}
+
+type TableRowProps = {
+  accountId: AccountId
+  accountNumber: number
+  feeAsset: Asset
+  toggleAccountId: (accountId: AccountId) => void
+}
+
+const disabledProp = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
+
+const TableRow = forwardRef<TableRowProps, 'div'>(
+  ({ feeAsset, accountId, accountNumber, toggleAccountId }, ref) => {
+    const handleChange = useCallback(() => toggleAccountId(accountId), [accountId, toggleAccountId])
+    const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
+    const balanceFilter = useMemo(
+      () => ({ assetId: feeAsset.assetId, accountId }),
+      [feeAsset, accountId],
+    )
+
+    const feeAssetBalancePrecision = useAppSelector(s =>
+      selectPortfolioCryptoPrecisionBalanceByFilter(s, balanceFilter),
+    )
+    const pubkey = useMemo(() => fromAccountId(accountId).account, [accountId])
+
+    return (
+      <Tr>
+        <Td>
+          <RawText>{accountNumber}</RawText>
+        </Td>
+        <Td>
+          <Switch onChange={handleChange} />
+        </Td>
+        <Td>
+          <Tooltip label={pubkey}>
```

Comment:
We don't want users to see their xpub directly IMO - they can't refer to it as "my account with these first four last four" since they would've never seen it before

Reply:
Nice catch. I've modifed the helper that gets the account label to provide an "un-shortened" version for the tooltip
c92637702b177e8ac5381c4bf5eb39e75457905e

Reply:
Not sure that makes sense, the row still looks off and the tooltip is effectively useless now
<img width="639" alt="image" src="https://github.com/shapeshift/web/assets/17035424/6e49500b-a39a-40c4-94fb-71b743d00f3f">

Reply:
<img width="679" alt="image" src="https://github.com/shapeshift/web/assets/17035424/b3c9fe98-d22c-41a0-9cad-bebca2932590">

2c4cead1d8ddab62de28f3de1476b90fa9cb4918